server/sonar-server/src/main/java/org/sonar/server/platform/web/RootFilter.java

```diff
@@ -50,26 +46,10 @@
  */
 public class RootFilter implements Filter {
 
-  private static final String CONFIG_SEPARATOR = ",";
-  private static final String URL_SEPARATOR = "/";
-
-  private static final String MESSAGE_WITH_QUERY = "%s %s?%s";
-  private static final String MESSAGE_WITHOUT_QUERY = "%s %s";
   public static final org.sonar.api.utils.log.Logger Logger = Loggers.get("http");
 
-  private String contextRoot;
-  private Set<String> staticResourceDirs;
-
   @Override
   public void init(FilterConfig filterConfig) throws ServletException {
```

Comment:
![CRITICAL](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-critical.png) Add a nested comment explaining why this method is empty, throw an UnsupportedOperationException or complete the implementation. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/sonarqube/coding_rules#rule_key=squid%3AS1186)